test/framework/cluster.go

```diff
@@ -537,6 +537,15 @@ func (e *ClusterE2ETest) ValidateCluster(kubeVersion v1alpha1.KubernetesVersion)
 	}
 }
 
+func (e *ClusterE2ETest) ValidateClusterNotUpgrading() {
+	ctx := context.Background()
+	e.T.Logf("Validating cluster %s not performing rolling upgrade", e.ClusterName)
+	err := e.KubectlClient.WaitForClusterReady(ctx, e.cluster(), "20s", e.ClusterName)
```

Comment:
i don't think this is the best way to verify machine not rollout. Instead we should compare the machines and see if they are same

Reply:
Changing the function to be `ValidateExistingMachines` might be more fitting then too